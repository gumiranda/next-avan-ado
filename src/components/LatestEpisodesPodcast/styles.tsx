import styled from 'styled-components';

export const Wrapper = styled.section`
  ul {
    list-style: none;
    display: grid;
    @media (max-width: 500px) {
      grid-template-columns: auto;
    }
    grid-template-columns: repeat(2, 1fr);
    gap: 1rem;

    li {
      background: var(--white);
      border: 1px solid var(--gray-100);
      padding: 2rem;
      border-radius: 2.1rem;
      position: relative;
      display: flex;
      align-items: center;
    }
  }
`;
export const EpisodeDetails = styled.div``;
