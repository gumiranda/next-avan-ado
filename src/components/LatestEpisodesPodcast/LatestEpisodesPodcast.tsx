import * as S from './styles';
interface IProps {
  propExample: any;
}
const LatestEpisodesPodcast = (props: IProps) => (
  <S.Wrapper>
    <h1>LatestEpisodesPodcast</h1>
  </S.Wrapper>
);

export default LatestEpisodesPodcast;
