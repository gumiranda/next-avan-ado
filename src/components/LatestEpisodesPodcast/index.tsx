import LatestEpisodesPodcast from './LatestEpisodesPodcast';
export default LatestEpisodesPodcast;
