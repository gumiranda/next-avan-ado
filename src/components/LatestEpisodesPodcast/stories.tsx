import { Story, Meta } from '@storybook/react/types-6-0';

import LatestEpisodesPodcast from './LatestEpisodesPodcast';

export default {
  title: 'LatestEpisodesPodcast',
  component: LatestEpisodesPodcast
} as Meta;

export const Basic: Story = (args) => (
  <LatestEpisodesPodcast
    episodeList={[]}
    title={'React AvanÃ§ado'}
    latestEpisodes={[]}
    {...args}
  />
);
