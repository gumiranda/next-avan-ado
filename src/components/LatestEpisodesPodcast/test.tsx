import { render, screen } from '@testing-library/react';

import LatestEpisodesPodcast from './LatestEpisodesPodcast';

describe('<LatestEpisodesPodcast />', () => {
  it('should render the heading', () => {
    const { container } = render(<LatestEpisodesPodcast />);
    expect(
      screen.getByRole('heading', { name: /LatestEpisodesPodcast/i })
    ).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
