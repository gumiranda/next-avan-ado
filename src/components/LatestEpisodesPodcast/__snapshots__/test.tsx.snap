// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LatestEpisodesPodcast /> should render the heading 1`] = `
<section
  class=""
>
  <h1>
    LatestEpisodesPodcast
  </h1>
</section>
`;
